gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-content-editor/api-documentation-v4-content-editor.component.ts

```diff
@@ -34,6 +34,7 @@ export class ApiDocumentationV4ContentEditorComponent implements ControlValueAcc
 
   preview = true;
   _value: string;
+  _disabled = false;
```

Comment:
is there a particular reason to have this `_` naming here ?
It feels like a private variable, but it's not

Reply:
Made it private and added a getter 👍